organizations/serializers.py

```diff
@@ -2,9 +2,18 @@
 Data layer serialization operations.  Converts querysets to simple
 python containers (mainly arrays and dicts).
 """
+from rest_framework import serializers
+
 from . import models
```

Comment:
Yep, and they are all wrong. You don't have to fix all of them; but, please correct those in the files you touch.